### PR TITLE
Fix android setContinuousRendering on init

### PR DESCRIPTION
**Pull request by hjanetzek**
Fixes https://github.com/tangrams/tangram-es/issues/706

### CODE REVIEW

File: android/tangram/src/com/mapzen/tangram/MapController.java [hunk 1]
```diff
@@ -123,35 +123,13 @@ private Bitmap capture() {
...
-        return new MapController(context, sceneFilePath);
-    }
-
-    /**
-     * Set the view in which the map will be drawn
-     * @param view GLSurfaceView where the map will be displayed; input events from this view will
-     * be handled by the resulting MapController
-     */
-    void setView(GLSurfaceView view) {
         touchInput = new TouchInput(view.getContext());
```
Comment (tallytalwar) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: I do not think we should allow to have touch handling if the view is not set. This change will allow for one.

**Comment by tallytalwar** [comment]
I think for now ... https://github.com/tangrams/tangram-es/pull/707 is has the minimal change, which will work.
A more API changing change for this will mean redoing a lot of API demo work.
We can talk more on this later, when we are done with SDC!

File: android/tangram/src/com/mapzen/tangram/MapController.java (hunk 1 above)
Comment (hjanetzek) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: I thought MapView is our external API now (setView is package visible only) and GLSurfaceView is  added to the Fragment in postExecute (again) so it cant get input before complete initialiazation

**Comment by hjanetzek** [comment]
ok

**Comment by tallytalwar** [comment]
This looks good to me.
@blair1618 it will be good for you to have a look at this one.

**Comment by blair1618** [comment]
Yeah this looks good, just update the Javadoc for `MapController()`. I would also prefer the new method be called `init()` to correspond to `nativeInit()` but that's a smaller detail.

**Pull request merged**